app/views/events/index.html.erb

```diff
@@ -6,6 +6,7 @@
         <th> <%= Event.human_attribute_name(:name) %> </th>
         <th><%= Event.human_attribute_name(:duration) %> </th>
         <th><%= Event.human_attribute_name(:started_at) %> </th>
+        <th><%= t('.status')%>
```

Comment:
Status should be translated the same like other event attributes. Check line above.